### PR TITLE
Mobile Header Fixes

### PR DESCRIPTION
**Pull request by evantahler**
A collection of fixes to make headers better on mobile.  See the comments below

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: components/docs/PluginsList.tsx [hunk 1]
```diff
@@ -5,7 +5,11 @@ import { PluginManifestItem } from "../../scripts/generate_plugin_json";
 
 const PluginDocsLink = (plugin: PluginManifestItem) => {
   if (!plugin.docsUrl) return <span>{plugin.name}</span>;
-  return <Link href={plugin.docsUrl}>{plugin.name}</Link>;
+  return (
+    <Link href={plugin.docsUrl.replace("https://www.grouparoo.com", "")}>
+      {plugin.name}
+    </Link>
+  );
```
Comment (evantahler) [inline components/docs/PluginsList.tsx:12]: Make these links relative, so they work on localhost too

File: pages/docs/integrations/grouparoo-bigquery.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "@grouparoo/bigquery"
+title: "@grouparoo / bigquery"
```
Comment (evantahler) [inline pages/docs/integrations/grouparoo-bigquery.mdx:2]: Adding a space between these longer titles helps with wrapping

Before:
Screen Shot 2021-12-20 at 11 15 59 AM

After:

Screen Shot 2021-12-20 at 11 22 51 AM

File: scss/grouparoo.scss [hunk 3]
```diff
@@ -6,6 +6,9 @@
 @import "~@fortawesome/fontawesome-svg-core/styles.css";
 @import "../public/css/prism";
 
+$mobile-display-breakpoint: 600px;
+$mobile-font-breakpoint: 800px;
```
Comment (evantahler) [inline scss/grouparoo.scss:10]: On tablets and phones, we want to decrease the font sizes a bit

Before
Screen Shot 2021-12-20 at 11 15 59 AM


After
Screen Shot 2021-12-20 at 11 16 52 AM

File: scss/grouparoo.scss [hunk 4]
```diff
@@ -32,6 +35,7 @@ h4,
 h5,
 h6 {
   font-family: $font-header;
+  word-break: break-word;
```
Comment (evantahler) [inline scss/grouparoo.scss:38]: In all cases, if a header doesn't fit, we don't want to expand the viewport

File: components/docs/PluginsList.tsx (hunk 1 above)
Comment (krishnaglick) [reply components/docs/PluginsList.tsx:12]: Could we make them relative at generation, or would that cause other issues?

File: scss/grouparoo.scss (hunk 3 above)
Comment (krishnaglick) [reply scss/grouparoo.scss:10]: We might want to investigate using `rem` over `px`, but I don't know if it would help handle this use-case.

**Review by krishnaglick — APPROVED** [review]

File: scss/grouparoo.scss (hunk 3 above)
Comment (evantahler) [reply scss/grouparoo.scss:10]: I *think* we would have the same problem - we want a bigger font on desktop than mobile, even defined relatively (rem)

File: components/docs/PluginsList.tsx (hunk 1 above)
Comment (evantahler) [reply components/docs/PluginsList.tsx:12]: I think we want the FQDN in the data file, as that's what builds our app manifest that core uses

**Pull request merged**